.gitignore

```diff
@@ -44,3 +44,4 @@ changelog.txt
 # Srg2Source causes import bugs on these patches. If you intentionnally change these, remove the ignore.
 /patches/minecraft/net/minecraft/client/renderer/ViewFrustum.java.patch
 /patches/minecraft/net/minecraft/data/BlockModelDefinition.java.patch
+*.xml
```

Comment:
Don't change the `.gitignore` file for no reason. If you need to ignore something in git, use `.git/info/exclude` so that is it not committed publicly.

Reply:
I was trying to not include eclipseformatter.xml in my commit. Am now going with 'discard changes' on that file.